src/instructlab/cli/process/prune.py

```diff
@@ -0,0 +1,54 @@
+# Third Party
+import click
+
+# First Party
+from instructlab import clickext
+from instructlab.defaults import ILAB_PROCESS_STATUS
+from instructlab.process.process import (
+    display_remove_list,
+    filter_process_with_conditions,
+    remove_process,
+)
+
+
+@click.command(name="prune")
+@click.option(
+    "--older",
+    type=click.INT,
+    default=7,
+    help="Remove completed processes older than the specified number of days (defautl: 7 days). Use 0 to remove all.",
```

Comment:
The `prune --help` already says that the command is to remove completed processes. Just describe the argument.

```
.tox/py311-unit/bin/ilab process prune --help
Usage: ilab process prune [OPTIONS]

  Remove all completed processes older than a number of days.

Options:
  --older INTEGER  Remove completed processes older than the specified number
                   of days (defautl: 7 days). Use 0 to remove all.
  -f, --force      Used to forcefully remove process. Use with caution as it
                   permanently deletes process without further confirmation.
  --help           Show this message and exit.
```